data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,471 @@ mission "Unfettered returning home"
+			`	After waiting for some time, you spot two hooded figures further down the beach making their way toward you, and you start to doubt whether or not this was a good idea. Even when they're close enough to speak, their faces somehow remain impossible to glimpse beneath the hoods. "Hello, <last>," one says to you. "We are sorry for the secrecy, but the truth is, we cannot afford to take risks. It is best for all if you do not know who we are, and we needed a secluded area to speak to you."`
+			choice
+				`	"Tell me more."`
+					goto "tell me"
+				`	"How do you know me?"`
+					goto know
+				`	(Run away.)`
+			`	They seem surprised at your sudden change of mind, but make no effort to stop you.`
+			label "train back"
+			`	Without any incidents, you return to the train station and buy a ticket back to the spaceport.`
+				decline
+			label know
+			`	The second one grins. "Any outsider landing on our homeworld must be well-known. We know you have provided various assistances to our kind."`
+			`	"Though do not think we are unaware of your skirmishes with the Unfettered," the first adds with a serious tone.`
+				to display
+					"reputation: Hai (Unfettered)" < -1250
+			label "tell me"
+			`	Suddenly, you hear a loud roar coming from the sea behind you, and the ground begins to shake.`
+			choice
+				`	(Turn around.)`
+			`	You see the waves become agitated and the wind rise as a shape surrounded by fumes emerges out of the water, as if an ancient sea monster was rising from the depths.`
+			`	But it is no monster. It's a ship, and its guns are pointed straight at you!`
+			label "try again"
+			scene "scene/emerging ladybug"
+			choice
+				`	(Run!)`
+				`	(Wait.)`
+					goto wait
+				`	(Try to contact my fleet for help.)`
+					to display
+						"total ships" > 1
+				`	(Shoot it!)`
+					goto foolish
+			`	As you try to react, the Hai closest to you firmly grabs your arm and shouts through the noise. "Wait! If this ship wanted to kill you, <last>, you would already be dead."`
+			label wait
+			`	You can barely hear anything over the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm. They turn to you, one more at ease than the other. "This ship means you no harm."`
+			choice
+				`	(Say nothing.)`
+					goto explain
+				`	"Why didn't you say anything before that happened?"`
+			`	The second one replies, "We could have, but where's the fun in that? We wanted to see how you would react.`
+			label explain
+			`	"This ship is our guarantee that nothing goes wrong, should you attempt something against us. After all, we are still not sure if we can trust you."`
+			`	The two Hai then take you to a nearby cave, showing you another ship. The first one speaks again. "That ship outside is not for sale. This ship, however, is a mighty relic of the past, intended for war from the times our empire used to control the stars - an Anomalocaris!"`
```

Comment:
```suggestion
			`	The two Hai then take you to a nearby cave. Crystals embedded in the walls cast a faint blue glow as you follow them, until you approach a large chamber where sunlight breaks through cracks in the walls, revealing another ship. The first Hai then speaks again. "That ship outside is not for sale. This ship, however, is a mighty relic of the past, intended for war from the times our empire used to control the stars - an Anomalocaris!"`
```
This is the main ship of the mission; it should get its own grandiose reveal, instead of just "The Hai take you to another ship".